_sass/components/_accordions.scss

```diff
@@ -0,0 +1,67 @@
+// Accordions
+//
+// Togglable accordion. Button is visible only in 
+// screen sizes smaller than medium
+//
+// Markup: 
+// <ul accordion>
+//    <li accordion-item accordion-open="true">
+//        <h3>Heading</h3>
+//        <button class="accordion-button" accordion-button></button>
+//        <div class="accordion-content">Accordion Content</div
+//    </li>
+// </ul>
+//
+// Styleguide components.accordions
+
+['accordion-item'] {
+  position: relative;
+
+  &['accordion-open'='true'] {
+    .accordion-button:after {
+      content: "\f077";
+    }
+
+    .accordion-content {
+      display: inline-block;
+    }
+  }
+
+  &['accordion-open'='false'] {
+    .accordion-button:after {
+      content: "\f078";
+    }
+    .accordion-content {
+
+      @include respond-to(medium-up) {
```

Comment:
Rule sets should be ordered as follows: `@extends`, `@includes` without `@content`, properties, `@includes` with `@content`, nested rule sets